### PR TITLE
[#26] Set appID and orgID for notifications

### LINKED ISSUE #26: Set appID and orgID for notifications
Setting up the env variables for the NotoficationsBB dependances

### LINKED ISSUE #59: [FEATURE]  Remove user data
**Is your feature request related to a problem? Please describe.**
Remove user data

### PR DESCRIPTION
**Pull request by stefanvit**
## Description
Adding the enviorment variables for appID and orgID used for the NotificationBB dependances.

**Resolves #26**

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

- [x] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
Set new env variables for appID and orgID

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ## Fixed
- - Fix docs [#21](https://github.com/rokwire/polls-building-block/issues/21)
+- Set appID and orgID for notifications [#26](https://github.com/rokwire/polls-building-block/issues/26)
+- Fix docs [#21](https://github.com/rokwire/polls-building-block/issues/21)
```
Comment (petyos) [inline CHANGELOG.md:10]: You you have this here?

**Review by petyos — CHANGES_REQUESTED** [review]
Hi @stefanvit , please look the feedback I left in https://github.com/rokwire/lms-building-block/pull/60 Here the situation is identical.

**Review by petyos — APPROVED** [review]
LGTM

**Review by petyos — APPROVED** [review]

**Review by shurwit — APPROVED** [review]
Thanks @stefanvit, LGTM!

**Pull request merged**